docs/custom_domains.rst

```diff
@@ -1,85 +1,98 @@
-FAQ <https://support.cloudflare.com/hc/en-us/articles/115000310832-Certification-Authority-Authorization-CAA-FAQ>`_.
+         $ dig docs.fabfile.org
+         ...
+         ;; ANSWER SECTION:
+         docs.fabfile.org.   7200    IN  CNAME   readthedocs.io.
 
-.. note::
+      .. note::
 
-    Some older setups configured a CNAME record pointing to ``readthedocs.org``
-    or another variation. While these continue to resolve,
-    they do not yet allow us to acquire SSL certificates for those domains.
-    Point the CNAME to ``readthedocs.io``, with no subdomain, and re-request a certificate
-    by saving the domain in the project admin (:guilabel:`Domains` >
-    :guilabel:`Edit Domain`).
+         Some older setups configured a CNAME record pointing to ``readthedocs.org`` or another variation.
+         While these continue to resolve,
+         they do not yet allow us to acquire SSL certificates for those domains.
+         Follow the new setup to have a SSL certificate.
 
-    If you change the CNAME record, the SSL certificate issuance can take about
-    one hour.
+      **Notes for Cloudflare users**
 
-.. important::
+      - If your domain has configured CAA records, please do not forget to include
+        Cloudflare CAA entries, see their `Certification Authority Authorization (CAA)
+        FAQ <https://support.cloudflare.com/hc/en-us/articles/115000310832-Certification-Authority-Authorization-CAA-FAQ>`__.
 
-    Due to a limitation, a domain cannot be proxied on Cloudflare
-    to another Cloudflare account that also proxies.
-    This results in a "CNAME Cross-User Banned" error.
-    In order to do SSL termination, we must proxy this connection.
-    If you don't want us to do SSL termination for your domain --
-    **which means you are responsible for the SSL certificate** --
-    then set your CNAME to ``cloudflare-to-cloudflare.readthedocs.io``
-    instead of ``readthedocs.io``.
+      - Due to a limitation,
+        a domain cannot be proxied on Cloudflare to another Cloudflare account that also proxies.
+        This results in a "CNAME Cross-User Banned" error.
+        In order to do SSL termination, we must proxy this connection.
+        If you don't want us to do SSL termination for your domain —
+        **which means you are responsible for the SSL certificate** —
+        then set your CNAME to ``cloudflare-to-cloudflare.readthedocs.io`` instead of ``readthedocs.io``.
+        For more details, see `this previous issue`_.
 
-    For more details, see this `previous issue`_.
+        .. _this previous issue: https://github.com/readthedocs/readthedocs.org/issues/4395
 
-    .. _previous issue: https://github.com/readthedocs/readthedocs.org/issues/4395
+   .. tab:: Read the Docs for Business
 
+      - Go the :guilabel:`Admin` tab of your project
+      - Click on :guilabel:`Domains`
+      - Enter your domain and click on :guilabel:`Add`
+      - Follow the steps shown on the domain page
```

Comment:
We should likely expand on this:

```suggestion
      - Follow the steps shown on the domain page. This will require adding 2 DNS records, one pointing your custom domain to our servers, and another allowing us to provision an SSL certificate. 
```